tests/components/hassio/test_handler.py

```diff
@@ -320,8 +320,8 @@ async def test_api_ingress_panels(
     ],
 )
 async def test_api_headers(
+    aiohttp_raw_server,  # 'aiohttp_raw_server' must be before 'hass'!
```

Comment:
Is this because the `aiohttp_raw_server` fixture must be torn down before `hass`? Why?

Reply:
> Is this because the `aiohttp_raw_server` fixture must be torn down before `hass`?

Yes

> Why?

We create our own `TCPConnector` which is only torn down on `EVENT_HOMEASSISTANT_CLOSE`. With it still open, the test hangs until the 15s timeout is reached.
https://github.com/home-assistant/core/blob/25a80cd46f0c51f675294f859ee797d9081989a8/homeassistant/helpers/aiohttp_client.py#L283-L295